src/data_inclusion/schema/modes_accueil.py

```diff
@@ -5,10 +5,10 @@ class ModeAccueil(EnhancedEnum):
     A_DISTANCE = (
         "a-distance",
         "À distance",
-        None,
+        "L'offre de service est délivrée à distance, par téléphone ou internet.",
     )
     EN_PRESENTIEL = (
         "en-presentiel",
         "En présentiel",
-        None,
+        "L'offre de service est délivrée en présentiel dans les locaux spécifiés par la structure.",
```

Comment:
```suggestion
        "Le service est proposé en présentiel, à l'adresse associée.",
```

Reply:
```suggestion
        "Le service est proposé en présentiel, à l'adresse associée à ce service.",
```

* même commentaires
* le service n'est pas nécessairement dans les locaux de la structure (e.g. un camion-épicerie solidaire)